<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAutomateForm</class>
 <widget class="QMainWindow" name="CreateAutomateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="variables_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Переменные:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statement_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Состояния:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="variables_list">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>261</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="statements_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="rules_list">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rules_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Правила:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_variable_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить переменную</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_variable_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить переменную</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_variable_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить переменную</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_statement_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить состояние</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_statement_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить состояние</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_statement_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить состояние</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_rule_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить правило</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_rule_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить правило</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_rule_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>510</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить правило</string>
    </property>
   </widget>
   <widget class="QLabel" name="environs_functions_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Функции окружения:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="environs_functions_list">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_environs_function_buton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить функцию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_environs_function_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить функцию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_environs_function_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить функцию</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="ReadyButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>0</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#ReadyButton {
border: none;
background-color: rgb(217, 217, 217);

}
#ReadyButton:hover:!pressed {
background-color: rgb(145, 145, 145);
}
#ReadyButton:pressed {
background-color: rgb(94, 94, 94);
}</string>
    </property>
    <property name="text">
     <string>Готово</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/Play.png</normaloff>:/images/Play.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="Help"/>
    <addaction name="aboutProgrammAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="clearAction">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="Close">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="aboutProgrammAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
